src/drivers/db0.ts

```diff
@@ -0,0 +1,84 @@
+import type { Database } from "db0";
+import { Driver } from "../types";
+import { defineDriver } from "./utils";
+
+type Dialect = "postgresql" | "mysql" | "sqlite";
+
+interface ResultSchema {
+  rows: Array<{
+    id: string;
+    value: string;
+    created_at: string;
+    updated_at: string;
+  }>;
+}
+
+export interface Db0DriverOptions {
+  database: Database;
+  dialect: Dialect;
+  table?: string;
+}
+
+const DRIVER_NAME = "db0";
+
+export default defineDriver((opts: Db0DriverOptions) => {
+  opts.table = opts.table || "storage";
+
+  const dialects: Partial<Record<Dialect, Partial<Driver>>> = {
+    mysql: {
+      setItem: async (key, value) => {
+        await opts.database
```

Comment:
Refactor: To improve maintenance, i would use an if condition inside main `setItem` to override behavior for `mysql` dialect